bootstrap

```diff
@@ -168,6 +168,21 @@ if [ ! -e $ssh_id_file ]; then
 fi
 export config_dir="$base_run_dir/config"
 export client_server_config_dir="$config_dir/client-server"
+
+echo "cpu-partitioning:"
+if [ "${cpu_partitioning}" == "1" ]; then
+    echo "Enabled"
+
+    environment="process"
+    if [ "${endpoint}" == "k8s" ]; then
+	environment="container"
```

Comment:
We don't need to worry about it now, but eventually we need to figure out what to do for kata and cnv.  endpoint will be "k8s", but osruntime will be kata or cnv.  However, this problem is bigger than just us, as we need the VM to be configured in such a way to provide isolation (unless we find a way to do it 100% at runtime).

Reply:
Yep.  I'm pretty much assuming that this point I'm just laying down a framework that will cover a few cases and we will have to expand on it as we add other endpoints/behaviors.